### PR TITLE
add GetCreditCard method

### PR DESCRIPTION
**Pull request by rizidoro**
This PR implements the GetCreditCard method on `CartaoProtegido.asmx` endpoint.

### CODE REVIEW

File: spec/credit_cards_spec.rb [hunk 1]
```diff
@@ -329,4 +329,93 @@
       end
     end
   end
+
+
+  describe 'GetCreditCard' do
+    context 'when the response is enabled' do
+      before do
+        allow(ResponseToggler).to receive(:enabled?).with('get_credit_card').and_return(true) end
```
Comment (bvicenzo) [inline spec/credit_cards_spec.rb]: The ```end``` could be in the above line, or this could be a block with ```{ }```

**Comment by bvicenzo** [comment]
:envelope_with_arrow: :8ball:

**Comment by antoniofilho** [comment]
:mag: :credit_card:

File: spec/credit_cards_spec.rb (hunk 1 above)
Comment (no-donut-for-you) [inline spec/credit_cards_spec.rb]:
```ruby 
before { ResponseToggler.enable('get_credit_card') }
```

File: spec/credit_cards_spec.rb [hunk 2]
```diff
@@ -329,4 +329,93 @@
...
+            </GetCreditCardResponse>
+          </soap:Body>
+        </soap:Envelope>
+        XML
+      end
+    end
+
+    context 'when the response is disabled' do
+      before do
+        allow(ResponseToggler).to receive(:enabled?).with('get_credit_card').and_return(false)
```
Comment (no-donut-for-you) [inline spec/credit_cards_spec.rb]:
```ruby
before { ResponseToggler.enable('get_sale') }
```

File: spec/credit_cards_spec.rb [hunk 3]
```diff
@@ -15,9 +15,7 @@
 
   describe 'SaveCreditCard' do
     context 'when the response is enabled' do
-      before do
-        allow(ResponseToggler).to receive(:enabled?).with('save_credit_card').and_return(true)
-      end
+      before { ResponseToggler.enable('save_credit_card') }
```
Comment (britto) [inline spec/credit_cards_spec.rb]: Hi guys! long time no see :smile:

Just passing by to warn about `ResponseToggler` :warning:. Calling `enable` in the tests will actually write to redis. Therefore, these settings changes will leak between specs. So, I recommend keeping the stubs as they were, for safer tests.

Comment (antoniofilho) [inline spec/credit_cards_spec.rb]: Thank @britto we will chance to use stub :smile:

**Pull request merged**